### PR TITLE
Add Django 4.1 to test matrix

### PR DESCRIPTION
**Pull request by lorddaedra**
Added support for Django 4.1

### CODE REVIEW

File: extra_views/__init__.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 from extra_views.dates import CalendarMonthView
 from extra_views.formsets import FormSetView, InlineFormSetView, ModelFormSetView
 
-__version__ = "0.14.0"
+__version__ = "0.15.0"
```
Comment (sdolemelipone) [inline extra_views/__init__.py:13]: We don't need to perform a release yet as there aren't any actual code changes. Could you revert this part?

**Review by sdolemelipone — CHANGES_REQUESTED** [review]
Looks good, just one change requested :-)

**Review by sdolemelipone — APPROVED** [review]
Thanks for the PR! 

@jonashaag I'll create a separate PR to address the failing black test so this is good to go.

**Comment by jonashaag** [comment]
Thanks a lot!

**Pull request merged**